<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TExportMissionsDialog</class>
 <widget class="QDialog" name="TExportMissionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exporting mission</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnPause">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnDelete">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSetting">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnClose">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tvMissions">
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
